content/about.md

```diff
@@ -90,10 +97,10 @@ Unity and the server software in Django.
 
 ## The future?
 
-I've started programming with C/C++, then switched to high-level languages and technologies.
-My plan is to go full circle and return to low-level - probably with Rust. I'll try to combine
-my Linux knowledge and low-level programming (backed by high-level prototyping and testing),
-to develop on embedded devices or to build high-performance server/desktop apps.
+I've learned the basics of programming with C, then switched to high-level languages and technologies.
+My current self development plan is built around Rust. I'm aiming to combine
```

Comment:
sounds strange, maybe 'is focused on Rust; I aim at combining

Reply:
Doesn't sound strange to me, I'll ask more people to settle this

Reply:
fair

Reply:
Alright, got a confirmation from @nuczlab, changed the line